substratest/assets.py

```diff
@@ -50,17 +60,60 @@ def get(self):
         return self._asset
 
 
-class _FutureMixin(abc.ABC):
+class ComputePlanFuture(BaseFuture):
+    def __init__(self, asset, session):
+        self._asset = asset
+        self._getter = session.get_compute_plan
```

Comment:
Just having a second thought about this:
- the `_asset` variable could be called directly `compute_plan`: it cannot be something else.
- the `_getter`, `_get_*` variables are not needed, we could directly store the input `session` and use it in the `wait()` and `get()` methods.

I think this code has been adapted from the already existing `Future` class and can be improved as it is dealing with only one type of asset.

Reply:
Done!